Development.podspec

```diff
@@ -0,0 +1,103 @@
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { :git => 'https://github.com/square/workflow.git', :tag => "v#{s.version}" }
+
+  s.ios.deployment_target = '11.0'
+  s.swift_version = '5.0'
+  s.dependency 'Workflow'
+  s.dependency 'WorkflowUI'
+  s.source_files = 'swift/Samples/Dummy.swift'
+
+  s.subspec 'Dummy' do |ss|
+  end
+
+  s.default_subspecs = 'Dummy'
+
+  dir = Pathname.new(__FILE__).dirname
+  snapshot_test_env = {
+    'IMAGE_DIFF_DIR' => dir.join('swift/FailureDiffs'),
+  }
+
+  s.scheme = { 
+    environment_variables: snapshot_test_env
+  }
+
+  s.app_spec 'SampleApp' do |app_spec|
+    app_spec.source_files = 'swift/Samples/SampleApp/Sources/**/*.swift'
+    app_spec.resources = 'swift/Samples/SampleApp/Resources/**/*.swift'
+    app_spec.pod_target_xcconfig = {
+      'INFOPLIST_FILE' => '${PODS_ROOT}/../../swift/Samples/SampleApp/Configuration/Info.plist'
+    }
+  end
+
+  s.test_spec 'WorkflowTesting' do |test_spec|
+    test_spec.requires_app_host = true
+    test_spec.dependency 'WorkflowTesting'
+    test_spec.source_files = 'swift/WorkflowTesting/Tests/**/*.swift'
+  end
+
+  # TODO: Disabled because app specs cannot increase the deployment target of the root
+  # To use, increase the deployment target of this spec to 13.0 or higher
+  #
+  # s.app_spec 'SampleSwiftUIApp' do |app_spec|
+  #   app_spec.ios.deployment_target = '13.0'
+  #   app_spec.dependency 'WorkflowSwiftUI'
+  #   app_spec.pod_target_xcconfig = {
+  #     'IFNFOPLIST_FILE' => '${PODS_ROOT}/../swift/Samples/SampleSwiftUIApp/SampleSwiftUIApp/Configuration/Info.plist'
+  #   }
+  #   app_spec.source_files = 'SampleSwiftUIApp/SampleSwiftUIApp/**/*.swift'
+  # end
+
+  s.app_spec 'SampleTicTacToe' do |app_spec|
+    app_spec.source_files = 'swift/Samples/TicTacToe/Sources/**/*.swift'
+    app_spec.resources = 'swift/Samples/TicTacToe/Resources/**/*'
+    app_spec.dependency 'BackStackContainer'
+  end
+
+  s.test_spec 'TicTacToeTests' do |test_spec|
+    test_spec.dependency 'Development/SampleTicTacToe'
+    test_spec.requires_app_host = true
+    test_spec.app_host_name = 'Development/SampleTicTacToe'
+    test_spec.source_files = 'swift/Samples/TicTacToe/Tests/**/*.swift'
+  end
+
+  s.app_spec 'SampleSplitScreen' do |app_spec|
+    app_spec.dependency 'SplitScreenContainer'
+    app_spec.source_files = 'swift/Samples/SplitScreenContainer/DemoApp/**/*.swift'
+  end
+
+  s.test_spec 'SplitScreenTests' do |test_spec|
+    test_spec.dependency 'SplitScreenContainer'
+    test_spec.dependency 'Development/SampleSplitScreen'
+    test_spec.app_host_name = 'Development/SampleSplitScreen'
+    test_spec.requires_app_host = true
+    test_spec.source_files = 'swift/Samples/SplitScreenContainer/SnapshotTests/**/*.swift'
+
+    test_spec.framework = 'XCTest'
+
+    test_spec.dependency 'iOSSnapshotTestCase'
+
+    test_spec.resources = 'swift/Samples/SnapshotTests/SplitScreenContainer/ReferenceImages_64'
```

Comment:
I don't think this is required.

Reply:
This is required, since we are now copying the resources into the bundle. We had discussed this change offline, but I can go back to using global paths if you prefer

Reply:
Extra context: we now only have the `IMAGE_DIFF_DIR` specified as an environment variable

https://github.com/square/workflow/pull/964/files#diff-fd7235628a2b67d5e50864050de83789R23

Reply:
Discussed offline, but we decided on using both methods so that:

1. When recording, it places the created reference images in a sane location
2. When running the tests from different contexts (ex. an app host vs. the test bundle itself) the  resources will always be available without naming conflicts. By default the name of the bundle is a component of the path